CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ This release has deprecations. Please read [DEPRECATION] entries and consult
 the [upgrade guide](https://github.com/grafana/agent/blob/main/docs/upgrade-guide/_index.md)
 for detailed information.
 
+- [FEATURE] (beta) Add app o11y integration. (@kpelelis)
```

Comment:
Can you add a note that this depends on integrations-next being enabled?

Reply:
82ddd252